tests/informatics/test_scores.py

```diff
@@ -37,6 +37,21 @@ def mei_score() -> EIScore:
     )
 
 
+@pytest.fixture
+def mev_score() -> EVScore:
+    """Build an MEVScore."""
```

Comment:
Let's use "ev" instead of "mev" here

Reply:
https://github.com/CitrineInformatics/citrine-python/pull/281